IFC4x3/Sections/Resource definition data schemas/Schemas/IfcProfileResource/Entities/IfcOpenCrossProfileDef/DocEntity.xml

```diff
@@ -13,6 +13,9 @@
 		<DocAttribute Name="Tags" UniqueId="e0750c4d-1efe-493d-9699-706d8d0bc7d6" DefinedType="IfcLabel" AttributeFlags="1" AggregationType="1" AggregationLower="2" AggregationUpper="?">
 			<Documentation></Documentation>
 		</DocAttribute>
+		<DocAttribute Name="OffsetPoint" UniqueId="4f5b213a-261b-4c67-b80c-cb8b72473faa" DefinedType="IfcCartesianPoint" AttributeFlags="1">
+			<Documentation>Optional cartesian point to nominate start of profile curve. The provided slopes and widths emerge from this point.</Documentation>
```

Comment:
```suggestion
			<Documentation>Optional Cartesian point to nominate the profile curve start. The provided slopes and widths emerge from this point. If no value is given, the profile initiates at the alignment.</Documentation>
```

Reply:
I can't seem to make a suggestion on top of your suggestion: I'd replace "alignment" with "(0,0) in the profile's coordinate system" or similar.